go.mod

```diff
@@ -18,15 +18,18 @@ require (
 require (
 	github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358 // indirect
 	github.com/docker/docker-credential-helpers v0.7.0 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
 	github.com/fxamacker/cbor/v2 v2.4.0 // indirect
 	github.com/go-asn1-ber/asn1-ber v1.5.4 // indirect
 	github.com/go-ldap/ldap/v3 v3.4.4 // indirect
 	github.com/golang-jwt/jwt/v4 v4.5.0 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/veraison/go-cose v1.0.0 // indirect
 	github.com/x448/float16 v0.8.4 // indirect
 	golang.org/x/crypto v0.9.0 // indirect
 	golang.org/x/mod v0.10.0 // indirect
 	golang.org/x/sync v0.2.0 // indirect
 	golang.org/x/sys v0.8.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
were this new dependencies added automatically when you ran `go mod tidy` ?

Reply:
@priteshbandi yes

based on some digging, it came from adding this:
https://github.com/notaryproject/notation/pull/687/files#diff-44385ed93d507652b8fe6326f87be189cf97111ba62a17b145bc415752b83daaR12

The `cobra` `go.mod` requires the `go-md2man` and `yaml.v3`:
https://github.com/spf13/cobra/blob/main/go.mod#L6-L9

The `go-md2man` `go.mod` requires `blackfriday`:
https://github.com/cpuguy83/go-md2man/blob/master/go.mod#L5